packages/next/test/fixtures/00-app-dir-segment-options/data-context.js

```diff
@@ -0,0 +1,3 @@
+import react from 'react'
+
+export const DataContext = react.createContext(null)
```

Comment:
Maybe newbie question: who uses this context?

Reply:
Good catch this can go, copied the entire fixture and overlooked that